Remotion/Web/Core/Res/HTML/DropDownMenu.js

```diff
@@ -412,6 +416,15 @@ function DropDownMenu_EndOpenPopUp (menuID, context, selectionCount, evt, itemIn
   // Only reposition if opened via titleDiv
   if (evt == null)
     _dropDownMenu_repositionTimer = setTimeout(repositionHandler, _dropDownMenu_repositionInterval);
+
+  if(!$menuButton.attr(_dropDownMenu_button_timestampAttributeKey))
```

Comment:
invert if please. makes easier to read if there is no not-operator in the condition.